docs/docs/guides/ecosystem/mistral.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 1
+hide_table_of_contents: true
+---
+
+# MistralAI
+
+Weave automatically tracks and logs LLM calls made via the [MistralAI Python library](https://github.com/mistralai/client-python), after `weave.init()` is called.
+
+## Setup
+
+1. Install the MistralAI Python library:
+   ```bash
+   pip install mistralai weave
+   ```
+
+2. Initialize Weave in your Python script:
+   ```python
+   import weave
+   weave.init("my_llm_project")
+   ```
+   :::note
+   We patch the mistral `chat_completion` method for you to keep track of your LLM calls.
+   :::
+
+3. Use the MistralAI library as usual:
+
+    ```python
+    from mistralai.client import MistralClient
+    from mistralai.models.chat_completion import ChatMessage
+
+    api_key = os.environ["MISTRAL_API_KEY"]
```

Comment:
Recommend adding an `import os` so that this code will immediately work if copied.

Reply:
Thanks =)